### PR TITLE
project: listeners management documentation page

### PR DESCRIPTION
**Pull request by GiovannaMonti**
## Description

This PR adds a new documentation page to describe the functionalities of the new Listeners section in the Design area.

## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

File: docs/development_suite/api-console/api-design/endpoints.md [hunk 1]
```diff
@@ -78,7 +78,14 @@ For the CRUD endpoint it's not possible to set an internal Rewrite. The Internal
...
 
-This card possibly includes 4 tabs: _Security_, _Documentation_, _Microservice Gateway_ and _Advanced_.
+From this section it is possible to configure: _Listeners_, _Security_, _Documentation_, _Microservice Gateway_ and _Advanced_ settings for the Endpoint.
+
+### Listeners
+
+![Endpoint settings card - Security](img/listeners/endpoint-configure-listeners.png)
+
+This section lets you define from which Listener your Endpoint should be exposed.  
+Keep in mind that it is not possible to leave an Endpoint not exposed by any Listener, thus every Endpoint must be exposed by _at least one_ Listener.
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/endpoints.md:88]:
```suggestion
Keep in mind that you must expose and Endpoint with _at least_ one Listener.
```

File: docs/development_suite/api-console/api-design/img/listeners/create-listener.png
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/img/listeners/create-listener.png:1]: this image is taken from a 4k screen which makes it hard to read, can we take it again from a smaller resolution srceen? Also change the name of hte project please

File: docs/development_suite/api-console/api-design/img/listeners/listeners-api-gateway-ports.png
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/img/listeners/listeners-api-gateway-ports.png:1]: can you take this screenshot from a smaller screen please?

File: docs/development_suite/api-console/api-design/listeners.md [hunk 2]
```diff
@@ -6,16 +6,71 @@ sidebar_label: Create Listeners
...
 
-A listener is an **entry point** for external connections directed to your cluster. Listeners are configured on your API Gateway, where they are mapped to a single port, and define rules to route incoming requests from external clients to a resource in your cluster, such as a specific microservice, exposed through endpoints.
+A Listener is a logical entity configured on your API Gateway that checks for incoming connection requests directed to your cluster.  
+Each listener is mapped to a single port and defines rules to route incoming requests from external clients to a resource in your cluster, such as a specific microservice, exposed through endpoints.
 
-The purpose of this page is to describe all the functionalities that you can find in the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Mia-Platform Console.
+This page describes the features of a Listener and how to configure these from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console.
+
+:::info
+Currently Listeners are available only to Projects configured to use an **Envoy API Gateway**.
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:15]:
```suggestion
Currently Listeners are only available to Projects using an **Envoy API Gateway**.
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 3]
```diff
@@ -6,16 +6,71 @@ sidebar_label: Create Listeners
...
+:::
 
 ## Create a Listener
 
-:::info
-The listeners feature is only supported in combination with an **Envoy API Gateway**, which needs to be instantiated on your Project.
+Follow this three steps in order to create the new Listener and make an Endpoint reachable from outside the Console:
+- create the new Listener resource
+- expose the Endpoint on the Listener
+- define a new Traefik IngressRoute for the newly exposed API Gateway port
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:23]: this depends on the ingress route you are using; I'd put this into a warning box that remembers the user that it may need to configure their cluster ingress to route traffic towards the newly defined listener port, e.g. using traefik you may want to add a new route in the IngressRoute resource.

File: docs/development_suite/api-console/api-design/listeners.md [hunk 4]
```diff
@@ -6,16 +6,71 @@ sidebar_label: Create Listeners
...
+
+### Defining the Listener 
+
+Creating a new Listener is as easy as clicking on the **Add listener** button and specifying its `name` and `port`. Both these two fields must be unique on your Project.  
+
+Optionally, you can define whether to:
+- **expose new endpoints on the listener** by deafult
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:30]:
```suggestion
- **expose new endpoints on the listener** by default
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 5]
```diff
@@ -6,16 +6,71 @@ sidebar_label: Create Listeners
...
+- **expose all existing endpoints** on the listener
+
+![create-listener](img/listeners/create-listener.png)
+
+Creating a new Listener means opening the relative container port on the Api Gateway too. This step is automated and in fact you can find the new port under the Container Ports section of the API Gateway.
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:35]:
```suggestion
Creating a new Listener means opening the relative container port on the API Gateway too. This step is automated and in fact you can find the new port under the Container Ports section of the API Gateway.  
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 6]
```diff
@@ -6,16 +6,71 @@ sidebar_label: Create Listeners
...
+<!-- TODO: update image with disabled buttons -->
+![listeners-api-gateway-container-ports](img/listeners/listeners-api-gateway-ports.png)
+
+### Exposing an Endpoint on the Listener
+
+Once the new Listener has been created you can select it from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) of an Endpoint.
+
+### Define the Kubernetes Ingress
+
+In order to receive incoming requests from the newly exposed API Gateway port, it is required to define the [Traefik IngressRoute](https://doc.traefik.io/traefik/routing/providers/kubernetes-crd/#kind-ingressroute) for the same port too.
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:47]: traefik is just one of the possibilities, this solely depends on the cluster ingress; so let's rephrase this as an example when using traefik (I'd collapse this info in the same warning box for the above comment)

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/development_suite/api-console/api-design/listeners.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+This page describes the features of a Listener and how to configure these from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console.
+
+:::info
+Currently Listeners are available only to Projects configured to use an **Envoy API Gateway**.
+:::
+
+## Create a Listener
+
+Follow this three steps in order to create the new Listener and make an Endpoint reachable from outside the Console:
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:20]:
```suggestion
Follow these three steps in order to create a new Listener and make an Endpoint reachable from outside the Console:
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+- create the new Listener resource
+- expose the Endpoint on the Listener
+- define a new Traefik IngressRoute for the newly exposed API Gateway port
+
+### Defining the Listener 
+
+Creating a new Listener is as easy as clicking on the **Add listener** button and specifying its `name` and `port`. Both these two fields must be unique on your Project.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:27]:
```suggestion
To create a new listener, just click the **Add Listener** button and specify its `name` and `port`. Both fields must be unique in the Project. 
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+- expose the Endpoint on the Listener
+- define a new Traefik IngressRoute for the newly exposed API Gateway port
+
+### Defining the Listener 
+
+Creating a new Listener is as easy as clicking on the **Add listener** button and specifying its `name` and `port`. Both these two fields must be unique on your Project.  
+
+Optionally, you can define whether to:
+- **expose new endpoints on the listener** by deafult
+- **expose all existing endpoints** on the listener
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:31]: For these two options, I would explain a little more extensively what they imply

File: docs/development_suite/api-console/api-design/listeners.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+Creating a new Listener means opening the relative container port on the Api Gateway too. This step is automated and in fact you can find the new port under the Container Ports section of the API Gateway.  
+Container Ports associated to Listeners cannot be deleted. To delete them firstly delete the Listener and then you can delete the port.
+
+<!-- TODO: update image with disabled buttons -->
+![listeners-api-gateway-container-ports](img/listeners/listeners-api-gateway-ports.png)
+
+### Exposing an Endpoint on the Listener
+
+Once the new Listener has been created you can select it from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) of an Endpoint.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:43]:
```suggestion
Once the new listener has been created, it can be selected from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the project.
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+  - # some already existing route here ...
+  - match: Host(`<put-a-host-here>.<your-domain-here>`) # e.g.: Host(`mobile-apis.your-domain.com`)
+    kind: Rule
+    services:
+      - name: api-gateway # the name of your API Gateway service
+        port: 8082 # the port just exposed
+
+```
+
+At this point
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:66]: There is an interrupted sentence

File: docs/development_suite/api-console/api-design/listeners.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:67]: I'd add a new paragraph for "Edit (/modify) a listener". Here, I would explain what implies a modification of listener also from a container port side.

File: docs/development_suite/api-console/api-design/listeners.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+## Delete a Listener
+
+To delete a listener just click on the delete action.
+
+:::warning
+Be aware that if the Listener is the only one exposing some endpoints, these will be deleted as well.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:73]:
```suggestion
Be aware that if some Endpoints are exposed only on the listener you want to delete, they will be deleted too.
```

File: docs/development_suite/api-console/api-design/endpoints.md [hunk 14]
```diff
@@ -78,7 +78,14 @@ For the CRUD endpoint it's not possible to set an internal Rewrite. The Internal
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/endpoints.md:78]: With the introduction of listeners management, in this documentation page, we have also to change the image and the information about endpoint creation. See the following paragraph https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/endpoints#details

Moreover, even info about required fields in endpoint creation has to be updated by adding listeners field info

File: docs/development_suite/api-console/api-design/listeners.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+- Expose the Endpoint on the Listener
+- Configure the [Kubernetes Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) to make the new exposed port reachable
+
+### Defining the Listener 
+
+To create a new listener, just click the **Add Listener** button and specify its `name` and `port`. Both fields must be unique in the Project. 
+
+Optionally, you can define whether to:
+- **expose new endpoints on the listener** by default
+- **expose all existing endpoints** on the listener
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:31]:
```suggestion
Optionally, during listener creation, you can define whether to:
- **expose all new endpoints that will be created on that listener** by default
- **expose all existing endpoints** on that listener
```

File: docs/development_suite/api-console/api-design/listeners.md (hunk 12 above)
Comment (danielemarostica) [reply docs/development_suite/api-console/api-design/listeners.md:83]: added

File: docs/development_suite/api-console/api-design/endpoints.md (hunk 14 above)
Comment (danielemarostica) [reply docs/development_suite/api-console/api-design/endpoints.md:78]: updated screenshots

File: docs/development_suite/api-console/api-design/img/listeners/create-listener.png
Comment (danielemarostica) [reply docs/development_suite/api-console/api-design/img/listeners/create-listener.png:1]: done

File: docs/development_suite/api-console/api-design/img/listeners/listeners-api-gateway-ports.png
Comment (danielemarostica) [reply docs/development_suite/api-console/api-design/img/listeners/listeners-api-gateway-ports.png:1]: done

File: docs/development_suite/api-console/api-design/listeners.md (hunk 3 above)
Comment (danielemarostica) [reply docs/development_suite/api-console/api-design/listeners.md]: made both here and description below more generic on Ingress configuration and added an example for traefik

File: docs/development_suite/api-console/api-design/listeners.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+
+## What is a Listener
+
+A Listener is a logical entity configured on your API Gateway that checks for incoming connection requests directed to your cluster.  
+Each listener is mapped to a single port and defines rules to route incoming requests from external clients to a resource in your cluster, such as a specific microservice, exposed through endpoints.
+
+This page describes the features of a Listener and how to configure these from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console.
+
+:::info
+Currently Listeners are only available to Projects using an **Envoy API Gateway**.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:15]:
```suggestion
Currently Listeners are only available for Projects using an **Envoy API Gateway**.
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+- Create the new Listener resource
+- Expose the Endpoint on the Listener
+- Configure the [Kubernetes Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) to make the new exposed port reachable
+
+### Defining the Listener 
+
+To create a new listener, just click the **Add Listener** button and specify its `name` and `port`. Both fields must be unique in the Project. 
+
+Optionally, during listener creation, you can define whether to:
+- **expose all new endpoints** that will be created on that listener by default
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:30]:
```suggestion
- **select the listener** by default when creating a new endpoint
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+- **expose all existing endpoints** on that listener
+
+![create-listener](img/listeners/create-listener.png)
+
+Creating a new Listener means opening the relative container port on the API Gateway too. This step is automated and in fact you can find the new port under the Container Ports section of the API Gateway.  
+:::caution
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:36]: better spacing

```suggestion

:::caution
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+Once the new Listener has been created, it can be selected from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the project.
+
+### Configure the Kubernetes Ingress Controller
+
+In order to receive incoming requests from the newly exposed API Gateway port, it is required to configure the [Kubernetes Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) to make the exposed port reachable.  
+
+:::info
+For example, if your Kubernetes cluster is configured to use [Traefik](https://doc.traefik.io/traefik/providers/kubernetes-ingress/) as Ingress Controller provider, you may need to define a new [IngressRoute](https://doc.traefik.io/traefik/providers/kubernetes-crd/).
+
+To do this, on the git repository of your Project locate the correct `.ingressroute.yaml` file for your environment and extend it with the new route and defining a new `host`.
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/listeners.md:54]: instead of reporting here this piece of doc could we link this page?

https://docs.mia-platform.eu/docs/paas/traefik#expose-an-endpoint

File: docs/development_suite/api-console/api-design/listeners.md [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+For example, if your Kubernetes cluster is configured to use [Traefik](https://doc.traefik.io/traefik/providers/kubernetes-ingress/) as Ingress Controller provider, you may need to define a new [IngressRoute](https://doc.traefik.io/traefik/providers/kubernetes-crd/).
+
+For more details on configuring a new IngressRoute, please refer to [this guide](/paas/traefik.md#expose-an-endpoint).
+:::
+
+## Edit a Listener
+
+From the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console it is possible to update settings of a Listener.
+
+Changes made to the port of the Listener will be reflected on the Container Ports of the API Gateway too.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:67]: If you modify a listener, are there changes to be reflected also on the ingress route?

**Review by albertotessarotto — APPROVED** [review]

File: docs/development_suite/api-console/api-design/listeners.md [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+Container Ports associated to Listeners cannot be deleted. To delete them firstly delete the Listener and then you can delete the port.
+:::
+
+![listeners-api-gateway-container-ports](img/listeners/listeners-api-gateway-ports.png)
+
+### Exposing an Endpoint on the Listener
+
+There are two ways to expose an Endpoint on the Listener:
+- selecting the desired Listener from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the project
+- selecting the desired Listener from the specific [Endpoints table](/development_suite/api-console/api-design/listeners.md#endpoints-overview).
```
Comment (MIA-Deltat1995) [inline docs/development_suite/api-console/api-design/listeners.md:53]: remember to fix the link if change the following title

File: docs/development_suite/api-console/api-design/listeners.md [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+
+
+### Listeners overview for Endpoints
+
+This table provides an overview of which Endpoint is exposed by which Listeners.  
+It is possible to edit this configuration by just selecting the desired Listener for the desired Endpoint.
```
Comment (MIA-Deltat1995) [inline docs/development_suite/api-console/api-design/listeners.md:59]:
```suggestion
The Endpoints tab contains a table provides an overview of which Endpoint is exposed by which Listeners.  
It is possible to edit this configuration by just selecting the desired Listener for the desired Endpoint. 
```

File: docs/development_suite/api-console/api-design/endpoints.md [hunk 23]
```diff
@@ -78,7 +78,14 @@ For the CRUD endpoint it's not possible to set an internal Rewrite. The Internal
...
 
-This card possibly includes 4 tabs: _Security_, _Documentation_, _Microservice Gateway_ and _Advanced_.
+From this section it is possible to configure: _Listeners_, _Security_, _Documentation_, _Microservice Gateway_ and _Advanced_ settings for the Endpoint.
+
+### Listeners
+
+![Endpoint settings card - Listeners](img/listeners/endpoint-configure-listeners.png)
+
+This section lets you define from which Listener your Endpoint should be exposed.  
+Keep in mind that you must expose and Endpoint with _at least_ one Listener.
```
Comment (MIA-Deltat1995) [inline docs/development_suite/api-console/api-design/endpoints.md:88]:
```suggestion
Keep in mind that you must expose an Endpoint with _at least_ one Listener.
```

**Review by MIA-Deltat1995 — CHANGES_REQUESTED** [review]

File: docs/development_suite/api-console/api-design/img/listeners/create-listener.png
[the remark this answers is not in the document]
Comment (albertotessarotto) [reply docs/development_suite/api-console/api-design/img/listeners/create-listener.png:1]: agree

File: docs/development_suite/api-console/api-design/listeners.md [hunk 24]
```diff
@@ -0,0 +1,96 @@
...
+To create a new listener, just click the **Add Listener** button and specify its `name` and `port`. Both fields must be unique in the Project. 
+
+Optionally, during listener creation, you can define whether to:
+- **set the listener as default**, so that it is pre-selected when creating new endpoints. This comes in handy if you think you will use the listener to expose many new endpoints. You can always de-select it if needed.
+- **expose all existing endpoints** on that listener
+
+![create-listener](img/listeners/create-listener.png)
+
+Creating a new Listener means opening the relative container port on the API Gateway too. This step is automated and in fact you can find the new port under the Container Ports section of the API Gateway.  
+
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:41]:
```suggestion
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 25]
```diff
@@ -0,0 +1,96 @@
...
+:::info
+Currently only Projects configured with an [**Envoy API Gateway**](/runtime_suite/envoy-api-gateway/overview.md) can access the Listeners feature.
+:::
+
+## What is a Listener
+
+A Listener is a logical entity configured on your API Gateway that checks for incoming connection requests directed to your Project namespace.  
+Each listener is mapped to a single port and defines rules to route incoming requests from external clients to a resource in your namespace, such as a specific microservice, exposed through endpoints.
+
+This page describes the features of a Listener and how to configure these from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:20]:
```suggestion
This page describes the features of a Listener and how to configure them from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) in Console.
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 26]
```diff
@@ -0,0 +1,96 @@
...
+
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:21]:
```suggestion
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 27]
```diff
@@ -0,0 +1,96 @@
...
+Container Ports associated to Listeners cannot be deleted. To delete them firstly delete the Listener and then you can delete the port.
+:::
+
+![listeners-api-gateway-container-ports](img/listeners/listeners-api-gateway-ports.png)
+
+### Exposing an Endpoint on the Listener
+
+There are two ways to expose an Endpoint on the Listener:
+- selecting the desired Listener from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the project
+- selecting the desired Listener from the specific [Endpoints table](/development_suite/api-console/api-design/listeners.md#endpoints-overview).
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:53]:
```suggestion
- selecting the desired Listener from the specific [Endpoints table](/development_suite/api-console/api-design/listeners.md#endpoints-overview) within Listeners section
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 28]
```diff
@@ -0,0 +1,96 @@
...
+:::caution
+Container Ports associated to Listeners cannot be deleted. To delete them firstly delete the Listener and then you can delete the port.
+:::
+
+![listeners-api-gateway-container-ports](img/listeners/listeners-api-gateway-ports.png)
+
+### Exposing an Endpoint on the Listener
+
+There are two ways to expose an Endpoint on the Listener:
+- selecting the desired Listener from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the project
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:52]:
```suggestion
- selecting the desired Listener from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the Project
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 29]
```diff
@@ -0,0 +1,96 @@
...
+- selecting the desired Listener from the specific [Endpoints table](/development_suite/api-console/api-design/listeners.md#endpoints-overview).
+
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:54]:
```suggestion
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 30]
```diff
@@ -0,0 +1,96 @@
...
+
+### Listeners overview for Endpoints
+
+This table provides an overview of which Endpoint is exposed by which Listeners.  
+It is possible to edit this configuration by just selecting the desired Listener for the desired Endpoint. 
+
+![listeners-api-gateway-container-ports](img/listeners/endpoints-listeners-table.png)
+
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:62]:
```suggestion
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 31]
```diff
@@ -0,0 +1,96 @@
...
+Changes made to the port of the Listener will be reflected on the Container Ports of the API Gateway too.  
+This means that changing the port of a Listener will also enable the relative port on the API Gateway. The old port of the API Gateway stays enabled and it is up to the user to disable it if needed. 
+
+:::warning
+Make sure to modify the Kubernetes Ingress Controller accordingly.
+:::
+
+![edit-listener](img/listeners/edit-listener.png)
+
+
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:87]:
```suggestion
```

File: docs/development_suite/api-console/api-design/listeners.md [hunk 32]
```diff
@@ -0,0 +1,96 @@
...
+![listeners-api-gateway-container-ports](img/listeners/listeners-api-gateway-ports.png)
+
+### Exposing an Endpoint on the Listener
+
+There are two ways to expose an Endpoint on the Listener:
+- selecting the desired Listener from the [Listener Settings section](/development_suite/api-console/api-design/endpoints.md#listeners) within the Endpoint detail page of the project
+- selecting the desired Listener from the specific [Endpoints table](/development_suite/api-console/api-design/listeners.md#endpoints-overview).
+
+
+### Listeners overview for Endpoints
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:56]:
```suggestion
### Endpoints list with their listener relations
```

File: docs/development_suite/api-console/api-design/listeners.md (hunk 27 above)
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/listeners.md:53]:
```suggestion
- selecting the desired Listener from the specific [Endpoints table](/development_suite/api-console/api-design/listeners.md#endpoints-list-with-their-listener-relations).
```
